src/App.vue

```diff
@@ -11,11 +12,33 @@
         <p>We're sorry, we're not able to retrieve this information at the moment, please try back later</p>
       </section>
 
-      <section v-else>
+      <section class="grid" v-else>
         <div v-if="loading">Loading...</div>
 
+        <div class="beer-filter">
```

Comment:
Beer-filter could be a new component? You are reusing the same style and only the field type if changing. Look into vue slots and how you could use this to make this styled component reusable.

Components are a great way to encapsulate style as well as features. The simple fact you create a new "BEM  component" tells you it could be it's own Vue component as well

Reply:
Really like this > Components are a great way to encapsulate style as well as features. 😄